src/platform/monitoring/DowntimeNotification/tests/DowntimeNotification.e2e.spec.js

```diff
@@ -2,8 +2,8 @@ const moment = require('moment');
 const E2eHelpers = require('../../../testing/e2e/helpers');
 const Timeouts = require('../../../testing/e2e/timeouts');
 
-const createMockEndpoint = require('../../../../platform/testing/e2e/mock-helpers');
-const FacilityHelpers = require('../../../../applications/facility-locator/tests/facility-helpers');
+const createMockEndpoint = require('platform/testing/e2e/mock-helpers');
+const FacilityHelpers = require('applications/facility-locator/tests/facility-helpers');
```

Comment:
I've wondered if apps should have a `public` directory or an `{appName}/public.js` file to help show that code is sharable with other apps, instead of just having apps be able to freely import from other apps without the other knowing.

Reply:
(Not that that's within scope of this PR at all)